### ISSUE TITLE
Shorthand rule ignores rules with "!important"

### ISSUE THREAD

**Issue by nschonni** [brigade/scss-lint#127]
When a rule uses `!important` (whether or not this is a good idea :wink:) like:

``` scss
margin: 0 0 0 0 !important;
```

it doesn't report the shorthand should be `margin: 0 !important;`

After removing the `!important`, it does correctly report the issue.
